terms.html

```diff
@@ -91,12 +100,23 @@
 specifications. This specification decouples the <a>identity provider</a>
 concept into two distinct concepts: the <a>issuer</a> and the <a>holder</a>.
   </dd>
-  <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
+  <dt><dfn data-lt="issuers|issuer's|issuer(s)|issuing|issued">issuer</dfn></dt>
+  <dd>
+A role a <a>party</a> can perform as it constructs a <a>verifiable credential</a> 
+from a set of <a>claims</a> it has <a>authored</a>, meta-data, and proofs,
+typically for the purpose of transferring that to a <a>holder</a>.
+  </dd>
+  <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
   <dd>
-A role an <a>entity</a> can perform by asserting <a>claims</a> about one or
-more <a>subjects</a>, creating a <a>verifiable credential</a> from these
-<a>claims</a>, and transmitting the <a>verifiable credential</a> to a
-<a>holder</a>.
+An <a>entity</a> that has 'a mind of its own', which means, amongst other things, that it autonomously (sovereignly)
```

Comment:
```suggestion
An <a>entity</a> that has 'a mind of its own', which means, among other things, that it 
```

Trying to simplify language here to prevent questions around "What doe autonomous" mean? Is an IoT device autonomous if it's executing on it's code? What about new AI systems? Are they autonomous and sovereign? (there is no clear answer today, these are actively being debated... and we should avoid dragging those debates into the group). I think what you are saying still holds without these words in the description.